src/views/Tree/registries.js

```diff
@@ -0,0 +1,48 @@
+import { DenoLand } from "./registries/DenoLand";
```

Comment:
These JS files could be moved to a directory, right in the `src` folder. Maybe `/utils`?